chat-plugins/info.js

```diff
@@ -2206,6 +2206,74 @@ var commands = exports.commands = {
 	smogdexhelp: ["/analysis [pokemon], [generation] - Links to the Smogon University analysis for this Pok\u00e9mon in the given generation.",
 		"!analysis [pokemon], [generation] - Shows everyone this link. Requires: + % @ # & ~"],
 
+	veekun: function (target, broadcast, user) {
+		if (!this.canBroadcast()) return;
+
+		var baseLink = 'http://veekun.com/dex/';
+
+		var pokemon = Tools.getTemplate(target);
+		var item = Tools.getItem(target);
+		var move = Tools.getMove(target);
+		var ability = Tools.getAbility(target);
+		var nature = Tools.getNature(target);
+		var atLeastOne = false;
+
+		// Pokemon
+		if (pokemon.exists) {
+			atLeastOne = true;
+			if (pokemon.isNonstandard) return this.sendReply(pokemon.species + ' is not a real Pokemon.');
```

Comment:
This check would also apply for abilities (Rebound/Persistent) and moves (Paleo Wave).